pages/api/products/list.ts

```diff
@@ -5,13 +5,10 @@ export default async function list(req: NextApiRequest, res: NextApiResponse) {
     try {
         const { accessToken, storeHash } = await getSession(req);
         const bigcommerce = bigcommerceClient(accessToken, storeHash);
-        // Optional: pass in API params here
-        const params = [
-            'limit=11',
-        ].join('&');
+        const { page, limit } = req.query;
 
-        const { data } = await bigcommerce.get(`/catalog/products?${params}`);
-        res.status(200).json(data);
+        const response = await bigcommerce.get(`/catalog/products?page=${page}&limit=${limit}`);
```

Comment:
this is probably good for now. If in the future we want to expand the params, we can also make use of URLSearchParams:
```jsx
const params = new URLSearchParams({ limit, page, search, somethingElse }).toString();
const response = await bigcommerce.get(`/catalog/products?${params}`);
```